src/com/owncloud/android/services/observer/FileObserverService.java

```diff
@@ -276,16 +276,27 @@ private void addObservedFile(OCFile file, Account account) {
      */
     private void addObservedFile(String localPath, Account account) {
         File file = new File(localPath);
-        String parentPath = file.getParent();
-        FolderObserver observer = mFolderObserversMap.get(parentPath);
-        if (observer == null) {
-            observer = new FolderObserver(parentPath, account, getApplicationContext());
-            mFolderObserversMap.put(parentPath, observer);
-            Log_OC.d(TAG, "Observer added for parent folder " + parentPath + "/");
+
+        if(file.isDirectory()) {
+            File[] children = file.listFiles();
```

Comment:
Might seem odd, since we only observe changes in files downloaded and we enforce a synchronization when a folder/file is 

But think in the next case:
1. Folder is set as available offline, and all its contents are correctly sync'ed.
2. A file is added in the server side into the the folder (via web UI or desktop client)
3. The user browses in the folder via the Android app, or a full account sync is triggered; the new file in the server side is downloaded
-- from this moment, the file should be monitored for local changes; but it's not, because it didn't exist when the folder was set as available offline.